shardingsphere-features/shardingsphere-db-discovery/shardingsphere-db-discovery-distsql/shardingsphere-db-discovery-distsql-parser/src/main/antlr4/imports/db-discovery/RDLStatement.g4

```diff
@@ -104,7 +104,7 @@ property
     ;
 
 discoveryTypeName
-    : IDENTIFIER
+    : IDENTIFIER | STRING
```

Comment:
Is `IDENTIFIER` suitable in discoveryTypeName?

Reply:
Before the documentation is adjusted, it is recommended not to remove IDENTIFIER support, otherwise users will be confused.
I plan to merge this PR and then make uniform adjustments to IDENTIFIER (including documentation).